### DISCUSSION TITLE
Pros and cons of having a linked server with 2 servers vs separate ds servers

### DISCUSSION THREAD

**Discussion by Yonny24** [VorlonCD/bi-aidetection#291] [Q&A]
I always had a custom model as a linked server and the standard model linked to this from the drop-down and ticked.

Last week I modified them and set them up as individual servers, nothing linked.

I found they process the images 30% faster or more when not linked.
Is there an advantage of having them linked?

**Comment by charredchar** [comment]
As per the description, linked servers will run the image through multiple servers to check for items. By nature it has to wait for the previous server to finish before the linked server can start processing. Using separate servers will not process the same image multiple times, this will mean that if you have only one image and it doesn't go through your custom model server then you might miss the thing you're looking for.
